docs/webhooks.md

```diff
@@ -8,47 +8,40 @@ The param `is_base64` used in function `validate` from module `spylib.utils.hmac
 to better convey the meaning of the parameter
 
 
+**Breaking Changes in version 0.7.0**: <br>
+The webhook features become available from `spylib.webhook` module as functions 
+instead of from the `Token` class as methods. 
```

Comment:
```suggestion
* The webhook features become available from `spylib.webhook` module as functions 
instead of from the `Token` class as methods. 
* The function `spylib.webook_is_valid` is now available at `spylib.webhook.validate`
```